### PR TITLE
New command `slcli image share`

### PR DESCRIPTION
**Pull request by edsonarios**
Issue: #1853
```
softlayer-python/ (issue1853) $ slcli image share 57065 --account-id 1234
Image template 57065 was shared to account 1234.
```

### CODE REVIEW

File: SoftLayer/CLI/image/share.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"""Get details for an image."""
```
Comment (BrianSantivanez) [inline SoftLayer/CLI/image/share.py:1]: Update this comment

Comment (edsonarios) [reply SoftLayer/CLI/image/share.py:1]: Done

**Comment by caberos** [comment]
I think in future is possible implement the share by name option
E.g slcli image share 123456 --name testname

File: tests/CLI/modules/image_tests.py [hunk 2]
```diff
@@ -53,3 +53,15 @@ def test_datacenter_remove(self):
     def test_datacenter_remove_fails(self):
         result = self.run_command(['image', 'datacenter', '100', '--remove'])
         self.assertEqual(2, result.exit_code)
+
+    def test_share(self):
+        result = self.run_command(['image', 'share', '123456', '--account-id', '654321'])
+        self.assert_no_fail(result)
+
+    def test_share_without_id(self):
```
Comment (caberos) [inline tests/CLI/modules/image_tests.py:61]: add a assert to error message

**Review by caberos — CHANGES_REQUESTED** [review]

**Review by allmightyspiff — APPROVED** [review]

**Review by caberos — APPROVED** [review]
LGTM

**Pull request merged**